### PR TITLE
[WIP] Add Cross-console nav to 3scale

### PR DESCRIPTION
**Pull request by christiemolloy**
**What this PR does / why we need it**:

Adds the cross-console navigation to the masthead. (It does not work yet, that will come in a later PR)

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/APPDUX-121?jql=text%20~%20%22cross-console%203scale%22

**Verification steps** 

1. Go to any page and test that the cross-console nav works / is style correctly.
2. There is no server-side functionality that works yet.

**Special notes for your reviewer**:

**Note:** how do I get the image to upload properly?

Screen Shot 2020-05-06 at 4 23 06 PM

Screen Shot 2020-05-06 at 4 23 15 PM

### CODE REVIEW

File: app/views/shared/provider/_header.html.slim [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
+= javascript_pack_tag 'CrossConsole'
 
-header.pf-c-page__header id="user_widget" class='Header' class=('Header--master' if current_account.master?) role='banner'
+header.pf-c-page__header id="user_widget" class='app-page__header pf-m-no-toggle' class=('Header--master' if current_account.master?) role='banner'
 
-  = link_to provider_admin_dashboard_path, class: 'pf-c-page__header-brand Header-link', title: 'Dashboard' do
+  = link_to provider_admin_dashboard_path, class: 'pf-c-page__header-brand-link Header-link', title: 'Dashboard' do
       .Header-logo class=('Header-logo--withIcon' if active_menu?(:main_menu, :dashboard))
         span Red Hat 3scale API Management
+        img[src="./../../../../assets/images/Logo-Red_Hat-Integration-A-Reverse-RGB.png"]
```
Comment (thomasmaas) [inline app/views/shared/provider/_header.html.slim:10]: we are currently using encoded SVG embedded in css: https://github.com/3scale/porta/blob/master/app/assets/stylesheets/provider/_header.scss#L17

**Comment by thomasmaas** [comment]
Hi @christiemolloy is this done? Seems the logic to display/hide the cross console navigation based on the type of install is missing? Would also be good to have a screenshot.

**Comment by dlabaj** [comment]
@christiemolloy You can lift the code from here to hide it. https://github.com/Apicurio/apicurito/pull/106

**Comment by christiemolloy** [comment]
Closing in favor of #2077

**Pull request closed without merging**